tests/module_custom2.py

```diff
@@ -3,7 +3,10 @@
 class MyModule(ModuleType):
     # No __metamodule_init__, to make sure that that's legal
 
-    class_attr = "foo"
+    def __getattr__(self, attr):
+        if attr == "class_attr":
+            return "foo"
+        raise AttributeError
```

Comment:
Could you add a comment here about why the test needs this getattr? Otherwise lgtm